rackcity/api/serializers/asset_serializers.py

```diff
@@ -50,8 +53,35 @@ class Meta:
             'rack_position',
             'owner',
             'comment',
+            'mac_addresses',
+            'power_connections',
         )
 
+    def get_mac_addresses(self, asset):
+        try:
+            ports = NetworkPort.objects.filter(asset=asset.id)
+        except ObjectDoesNotExist:
+            return
+        mac_addresses = {}
+        for port in ports:
+            if port.mac_address:
+                mac_addresses[port.port_name] = port.mac_address
+        return mac_addresses
+
+    def get_power_connections(self, asset):
+        try:
+            ports = PowerPort.objects.filter(asset=asset.id)
+        except ObjectDoesNotExist:
+            return
+        power_connections = {}
+        for port in ports:
+            if port.power_connection:
+                power_connections[port.port_name] = {
+                    "left_right": port.power_connection.left_right,
+                    "port_number": port.power_connection.port_number
+                }
+        return power_connections  # YOU ARE HERE! Test with one that has connections
```

Comment:
I WAS HERE! And I did test with one that has connections. I forgot to delete the comment but I deleted it in my next PR